crates/ruff_linter/src/codes.rs

```diff
@@ -779,9 +779,9 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Flake8Pyi, "055") => (RuleGroup::Stable, rules::flake8_pyi::rules::UnnecessaryTypeUnion),
         (Flake8Pyi, "056") => (RuleGroup::Stable, rules::flake8_pyi::rules::UnsupportedMethodCallOnAll),
         (Flake8Pyi, "058") => (RuleGroup::Stable, rules::flake8_pyi::rules::GeneratorReturnFromIterMethod),
-        (Flake8Pyi, "057") => (RuleGroup::Preview, rules::flake8_pyi::rules::ByteStringUsage),
+        (Flake8Pyi, "057") => (RuleGroup::Stable, rules::flake8_pyi::rules::ByteStringUsage),
```

Comment:
I think this one has been around for a little less than 90 days -- it was merged on May 29 and released on May 31. But it's also a very simple and uncontroversial rule (IMO), so I'm okay with stabilising it a couple of weeks before we normally would 😄 it's not in our versioning policy that we have to wait 90 days

Reply:
Lol, that's correct. I only saw *May* and was like, that's a long time ago. I did do the proper math for the other rules.